syntax = "proto3";

package Ydb.Debug.V1;
option java_package = "com.yandex.ydb.debug.v1";

service DebugService {
    rpc PingPlainGrpc(PlainGrpcRequest) returns (PlainGrpcResponse);
}

message PlainGrpcRequest {
}

message PlainGrpcResponse {
    uint64 CallBackTs = 1;
}